docs/FEATURES-ROADMAP.md

```diff
@@ -0,0 +1 @@
+# Features roadmap
```

Comment:
Add a little description for such module.

Reply:
This doc was rremoved